packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -641,14 +719,18 @@ export default class ConversationService {
     payloadBundle: PayloadBundleOutgoingUnsent
   ): Promise<PayloadBundleOutgoing> {
     switch (payloadBundle.type) {
-      case GenericMessageType.ASSET: {
+      case GenericMessageType.ASSET:
+      case GenericMessageType.IMAGE: {
         if (payloadBundle.content) {
           if ((payloadBundle.content as ImageAssetContent).image) {
             return this.sendImage(conversationId, payloadBundle);
           }
-          throw new Error(`No send method implemented for sending other assets than images.`);
+          if ((payloadBundle.content as FileAssetContent).asset) {
+            return this.sendFile(conversationId, payloadBundle);
+          }
+          throw new Error(`No send method implemented for this kind of asset.`);
```

Comment:
Here we should also log a `payloadBundle.type` or something similar to specify "this kind of asset".

Reply:
And `payloadBundle.type` will be just `asset`.